woocommerce/wc-admin
# Add Razorpay to payment task for stores in India

Fixes #5764.

This PR adds Razorpay to the Payments task for stores that are in India.

Note: the copy and logo were improvised by me - we're still awaiting official assets.

### Screenshots

![Capture 2020-12-01 at 08 21 12](https://user-images.githubusercontent.com/63922/100760080-61f2bc00-33bf-11eb-88b7-b8dcdbf94918.gif)


### Detailed test instructions:

- Set up a new store
- Go through onboarding wizard, specify a location in India
- Enter the payments task from the setup checklist on the home screen
- Verify Razorpay is an option
- Set up Razorpay
- Verify the Razorpay plugin was installed, and that the gateway is enabled with the Key values specified (WooCommerce > Settings > Payments > Razorpay > Manage)



### Changelog Note:



Enhancement: add Razorpay payment gateway setup for stores in India.
